test/integration/groups_test.go

```diff
@@ -60,23 +60,14 @@ func TestBasicUserBasedGroupManipulation(t *testing.T) {
 		t.Errorf("expected %v, got %v", expectedClusterAdminGroups, clusterAdminUser.Groups)
 	}
 
-	valerieGroups := []string{"theGroup"}
-	firstValerie.Groups = append(firstValerie.Groups, valerieGroups...)
-	_, err = clusterAdminUserClient.Users().Update(firstValerie)
+	theGroup := &userapi.Group{}
+	theGroup.Name = "theGroup"
+	theGroup.Users = append(theGroup.Users, "valerie")
+	_, err = clusterAdminUserClient.Groups().Create(theGroup)
 	if err != nil {
 		t.Errorf("unexpected error: %v", err)
 	}
 
-	// make sure that user/~ returns system groups for backed users when it merges
```

Comment:
We still want this check, just look for `[]string{"system:authenticated", "system:authenticated:oauth"}` only.